File: Assignment3638/program13.c
//write a recursive program which accept string from user and count number of small characters
// Input : HElloWorID
// OutPut : 5

#include<stdio.h>

int SmallChar(char *str)
{
   static int iCount = 0;
    if(*str != '\0')
    {
        if((*str >= 'a') && (*str <= 'z'))
        {
            iCount++;
        }
        str++;
        SmallChar(str);
    }
    return iCount;
}
int main()
{
    char cValue[30];
    int iRet = 0;

    printf("Enter the String..\n");
    scanf(" %[^'\n']s",cValue);

    iRet = SmallChar(cValue);
    printf("The Count of Small Characters is : %d\n",iRet);

    return 0;
}

File: Assignment47&50/program13.java
//write java program which accept number from user and return the count digits in between 3 and 7

import java.lang.*;
import java.util.*;

class Digit
{
    public int CountRange(int iNo)
    {
        int idigit = 0, iCnt = 0;
        int i = 0;  

        if(iNo == -iNo)
        {
            iNo = -iNo;
        }
        while(iNo != 0)
    {
        idigit = iNo % 10;
        if(idigit > 3 &&  idigit < 7)
        {
            iCnt++;
        }
        iNo = iNo/10;
    }
        return iCnt;
    }
   
}


class program13
{
    public static void main(String arg[])
    {
        
        int iNo = 0;
        
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the Number..");
        iNo = sobj.nextInt();

        Digit dobj = new Digit();
        int iRet = dobj.CountRange(iNo);
        System.out.println("count of Number in between 3 and 7 : "+iRet);
        
    }
}